webwhatsapi/js/wapi.js

```diff
@@ -13,6 +13,7 @@ if (!window.Store) {
             let neededObjects = [
                 { id: "Store", conditions: (module) => (module.Chat && module.Msg) ? module : null },
                 { id: "Wap", conditions: (module) => (module.createGroup) ? module : null },
+                {id: "MediaCollection", conditions: (module) => (module.prototype && module.prototype.processFiles !== undefined) ? new module() : null},
```

Comment:
Memory collection is a simple class. You could instance it more than once...

Reply:
Only instance once.. without >new module()< don't work

Reply:
{id: "MediaCollection", conditions: (module) => (module.prototype && module.prototype.processFiles !== undefined) ? module : null}
........

var mc = new Store.MediaCollection();

Reply:
ooooh.. i see, now i understand what you say!